tests/integration/local/start_api/test_start_api.py

```diff
@@ -1629,3 +1634,96 @@ def test_swagger_got_parsed_and_api_is_reachable_and_payload_version_is_2(self):
         self.assertEqual(response_data.get("version", {}), "2.0")
         self.assertIsNone(response_data.get("multiValueHeaders"))
         self.assertIsNotNone(response_data.get("cookies"))
+
+
+class TestWarmContainersBaseClass(StartApiIntegBaseClass):
+    @classmethod
+    def setUpClass(cls):
+        # This is the directory for tests/integration which will be used to file the testdata
+        # files for integ tests
+        template = cls.integration_dir + cls.template_path
```

Comment:
Isn't this controlled by `StartApiIntegBaseClass`? StartApiIntegBaseClass is starting up a `start-api`

Reply:
refactored this part